package.json

```diff
@@ -17,8 +17,8 @@
     "deep-freeze": "0.0.1",
     "js-yaml": "3.11.0",
     "lodash": "4.17.10",
-    "patternfly": "3.45.3",
-    "patternfly-react": "2.3.0",
+    "patternfly": "3.46.1",
```

Comment:
Updating a lot of npm packages that are not related to this `yarn` PR. Or are they?